development/webpack/utils/plugins/ManifestPlugin/helpers.ts

```diff
@@ -24,6 +26,20 @@ export function transformManifest(args: { lockdown: boolean; test: boolean }) {
     }
   }
 
+  /**
+   * This function sets predefined flags in the manifest's _flags property.
+   *
+   * @param browserManifest - The Chrome extension manifest object to modify
+   */
+  function addManifestFlags(browserManifest: chrome.runtime.Manifest) {
+    browserManifest._flags = manifestFlags;
+  }
+
+  // Add manifest flags only for non-test builds so the test build is not affected by local feature flags
+  if (!args.test) {
```

Comment:
Perhaps we should use `args.env === 'development'` here, so that the transform only gets used when in development

Reply:
Addressed cc224d78f239296fde074d55608646da94a7c902